### ISSUE TITLE
Select new password - onblur not triggering reevaluate

### ISSUE THREAD

**Issue by georules** [google/exposure-notifications-verification-server#740]
### TL;DR
Onblur not triggering reevaluate when using mouse right click paste.  This happens from users using password generators and copy-pasting.

**Expected behavior**
Users expect set password button to become active & password validation to be reevaluated on all changes to password field

**Observed behavior**
Onblur not triggering reevaluate when using mouse right click paste

### Reproduction
Make password in some other editor
paste it with context mouse menu into select new password to password and retype
observe set password is not changed

**Screenshots**
https://drive.google.com/file/d/1RnHGfG9Ir5K2eCOBpBP3OMjaiLLwpVXE/view?usp=sharing

**Environment**

- OS: Macos
- Browser: Chrome
- Version:Version 85.0.4183.121 (Official Build) (64-bit)

**Additional information**

**Comment by whaught** [comment]
Oops sorry @jeremyfaller - this is done
https://github.com/google/exposure-notifications-verification-server/pull/741

/close
